packages/client/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@webcomponents/internal-site-client",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Client JavaScript used on webcomponents.org",
+  "author": "Google LLC",
+  "license": "Apache-2.0",
+  "type": "module",
+  "scripts": {
+    "build": "wireit",
+    "build:dev": "wireit",
+    "build:prod": "wireit",
+    "check:types": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:dev",
+        "build:prod",
+        "check:types"
+      ]
+    },
+    "build:dev": {
```

Comment:
Could we have a command that does `build:dev` and `check:types`, but not `build:prod`?

Reply:
What would you call it?

Reply:
I'd be tempted to call it `build:dev` and this one `build:ts:dev` with tests depending on `build:dev`.

Reply:
I see this also in #1335 ✔️